### PR TITLE
Remove Switch, and making `across the guardian` the default

### PR DESCRIPTION
**Pull request by NataliaLKB**
The results of the test have come back that having the **across the guardian** section as the first tab in the Most Popular container did perform _slightly_ better. This was statistically significant.

This PR removes the switch, and makes **across the guardian** permanently first.

### CODE REVIEW

File: common/app/views/fragments/collections/popular.scala.html [hunk 1]
```diff
@@ -7,76 +7,42 @@
...
-                            <a href="#tabs-popular-@info.rowNum" data-link-name="tab @info.rowNum @section.heading"><span class="u-h">Popular </span>@Html(Localisation(section.heading.stripPrefix("popular ")))</a>
-                        </li>
-                        }
-                    } else {
-                        @popular.zipWithRowInfo.map{ case (section, info) =>
-                        <li class="tabs__tab@if(info.isFirst){ tabs__tab--selected tone-colour tone-accent-border}" role="tab" id="tabs-popular-@info.rowNum-tab"@if(info.isFirst){ aria-selected="true"} aria-controls="tabs-popular-@info.rowNum">
-                            <a href="#tabs-popular-@info.rowNum" data-link-name="tab @info.rowNum @section.heading"><span class="u-h">Popular </span>@Html(Localisation(section.heading.stripPrefix("popular ")))</a>
-                        </li>
-                        }
+                    @popular.zipWithRowInfo.map{ case (section, info) =>
```
Comment (gtrufitt) [inline common/app/views/fragments/collections/popular.scala.html]: Just checking that you didn't want `@popular.reverse.` as that was the test that you're making default now, right?

**Comment by gtrufitt** [comment]
:+1: :cool:

File: common/app/views/fragments/collections/popular.scala.html (hunk 1 above)
Comment (NataliaLKB) [inline common/app/views/fragments/collections/popular.scala.html]: Hey thanks for checking this! I thought this was a better way of doing it: https://github.com/guardian/frontend/blob/remove-pop-test/onward/app/controllers/MostPopularController.scala#L34

Comment (gtrufitt) [inline common/app/views/fragments/collections/popular.scala.html]: Ah yes, makes sense :+1:

**Pull request merged**